abi/src/main/java/org/web3j/abi/EventEncoder.java

```diff
@@ -20,21 +20,21 @@
     private EventEncoder() { }
 
     public static String encode(Event function) {
-        List<TypeReference<Type>> indexedParameters = function.getIndexedParameters();
         List<TypeReference<Type>> nonIndexedParameters = function.getNonIndexedParameters();
+        List<TypeReference<Type>> indexedParameters = function.getIndexedParameters();
```

Comment:
Why is this change required?

Reply:
It's just to make the re-ordering look consistent throughout the EventEncoder class. Since I changed from placing indexParameters then unindexedParameter in the following code, 

`-        List<TypeReference<T>> parameters = new ArrayList<>(indexParameters);
 -        parameters.addAll(nonIndexedParameters);
 +        List<TypeReference<T>> parameters = new ArrayList<>(nonIndexedParameters);
 +        parameters.addAll(indexParameters);`

I thought it would be good to also change rest of the class to follow the same suit (nonindexedParameters then indexedParameters)